meadServer/client/src/App.js

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
In getNames, you should try to add your states all at once instead of queueing multiple state transitions